README.md

```diff
@@ -13,13 +13,35 @@ To build and install the latest version in a running AEM instance, simply do
 ```
 mvn clean install sling:install
 ```
+
 This installs everything by default to `localhost:4502` without any context path. You can also configure the install location with the following maven properties:
-* `aem.host`: the name of the AEM instance
-* `aem.port`: the port number of the AEM instance
-* `aem.contextPath`: the context path (if any) of your AEM instance, starting with `/`
+
+-   `aem.host`: the name of the AEM instance
+-   `aem.port`: the port number of the AEM instance
+-   `aem.contextPath`: the context path (if any) of your AEM instance, starting with `/`
 
 ## Using the GraphQL client
 
+### Prerequisites
+
+The GraphQL client needs a service user to read the configurations, so consumers must provide one or more user mappings. The subservice name is `graphql-client-configuration` and the minimum privilege matrix for the service user(s) is the following:
```

Comment:
We should also explain **where** the user mappings should be defined in the customer project, and probably even provide an example config in the archetype. Most AEM developers don't do this very often, so we have to make sure it's 100% clear what needs to be done by customers to get this working.

I'm not entirely sure, but it can also be that this is going to be different on AEMasCS and on AEM on-prem, so we should really provide very precise guidance on how to configure this.